### ISSUE TITLE
FileNotFoundError: [Errno 2]

### ISSUE THREAD

**Issue by regilim** [Open-Debin/Emotion-FAN#3]
FileNotFoundError: [Errno 2] No such file or directory: './Data/Train/Fear/013736360.avi' ? I noticed there's no folder named Train in Data folder

**Comment by Open-Debin** [comment]
Yes, we are no right to publish the dataset. If you want the AFEW dataset you can send an apply email to the official. The email is abhinav.dhall@monash.edu. PS: Could you please star my project?

**Comment by rshivansh** [comment]
even after downloading the data I get the same error.

**Comment by Open-Debin** [comment]
Please notice  1、your data path in disk. 2、your data path in the code3、data list  4 、the relevant code at DebinMeng_train.py
error only at these 4 points.

Also, I give an idea. Maybe your data path in the disk is ./Data/Train/Fear/013736360 rather than ./Data/Train/Fear/013736360.avi.  '1.’ and '2.' should be consistent.

Plus, could you please give me a star? thanks!!

**Comment by rshivansh** [comment]
hi, shouldn't the data be like ./Data/Train/Fear/013736360/ 1.jpg 2.jpg 3.jpg (these are the frames).

I have starred.

**Comment by LeiiY** [comment]
> hi, shouldn't the data be like ./Data/Train/Fear/013736360/ 1.jpg 2.jpg 3.jpg (these are the frames).
> 
> I have starred.

how u convert this video into img？ frame by frame？

**Comment by Open-Debin** [comment]
You can use ffmpeg our Opencv to convert video into image. Many instructions online. Give me a star, thanks. @LeiiY

**Comment by LeiiY** [comment]
> You can use ffmpeg our Opencv to convert video into image. Many instructions online. Give me a star, thanks. @LeiiY

so after convert to img，should i detect faces and cut  them into 224*224 or just start trainning？i used dlib，but  failed detecting faces in many frames

**Comment by mahdirayeni** [comment]
When running the following code:

`with open(video_list, 'r') as imf:
        index = []
        for id, line in enumerate(imf):

            video_label = line.strip().split()

            video_name = video_label[0]  # name of video
            label = rectify_label[video_label[1]]  # label of video

            video_path = os.path.join(video_root, video_name)  # video_path is the path of each video
            ###  for sampling triple imgs in the single video_path  ####

            img_lists = os.listdir(video_path)
            img_lists.sort()  # sort files by ascending
            img_count = len(img_lists)  # number of frames in video
            num_per_part = int(img_count) // 3`

I get the following error:
`FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:/Users/m_rayeni/Emotion-FAN/Data/Train/Fear/013736360'`

So what is wrong here?

**Comment by Open-Debin** [comment]
@LeiiY @rshivansh @regilim @mahdirayeni 
Merry Christmas, I recently update the Emotion-FAN, new features include data process, environment install, CK+ code, Baseline code, and more detail instructions. Also, you can find the old version directory of Emotion-FAN in the README.md. I hope my new updates can help you greatly. Please see the Emotion-FAN for more details.
